ldms/src/contrib/sampler/variorum_sampler/variorum_sampler.c

```diff
@@ -0,0 +1,234 @@
+#define _GNU_SOURCE
+#include <inttypes.h>
+#include <unistd.h>
+#include <sys/errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "ldms.h"
+#include "ldmsd.h"
+#include "sampler_base.h"
+#include "variorum.h"
+#include "jansson.h"
+#include "variorum_topology.h"
+
+#define SAMP "variorum_sampler"
+
+static ldms_set_t set = NULL;
+static ldmsd_msg_log_f msglog;
+static base_data_t base;
+static int nsockets;
+static const char *SOCKET_METRICS[] = {"power_cpu_watts_socket_", "power_gpu_watts_socket_", "power_mem_watts_socket_"};
+static char** metric_names = NULL;
+static int i_node;
+static int i_sock;
+static int i_cpu;
+static int i_gpu;
+static int i_mem;
+static int lh_idx;
+static ldms_mval_t* rec_idxs;
+static char* result_string;
+
+static int create_metric_set(base_data_t base)
+{
+	int rc, i, metric, socket;
```

Comment:
This file needs to be reformatted to match the rest of ovis-ldms. For instance:

Indent is 8 spaces, no tabs